### DISCUSSION TITLE
There is an obvious difference to calcaulate thermal conductivity with different deepmd version

### DISCUSSION THREAD

**Discussion by WU2JING** [deepmodeling/deepmd-kit#1882] [Q&A]
Now, my team try to calcaulate the thermal conductivity of crystal materials with EMD method. However, when same training data and input file was adopted to train a model, there is a big difference for results between 2.0.0.b3 and later versions, as follow figures.

To avoid uncertainty，we  have tested many times, the result same with follow figures.

![image](https://user-images.githubusercontent.com/68996784/187424205-06680e0a-b92d-4b4f-aff3-35d3db7a3de7.png)
![image](https://user-images.githubusercontent.com/68996784/187424298-509300af-64c9-4669-afb0-a6b6a1a16100.png)

**Comment by WU2JING** [comment]
Can you help me.
Any discussion is welcomed.

**Comment by wanghan-iapcm** [comment]
The issue in the calculation of heat flux is fixed since v2.0.1, please see #1093 and #1451

**Reply by WU2JING** [reply]
Ok，thank you.
